### PR TITLE
update import paths to "github.com/graphql-go/graphql" and "github.com/graphql-go/graphql-relay-go"

### PR DESCRIPTION
**Pull request by pyros2097**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # graphql-relay-go
 
-A Go/Golang library to help construct a [graphql-go](https://github.com/chris-ramon/graphql-go) server supporting react-relay.
+A Go/Golang library to help construct a [graphql-go](https://github.com/graphql-go/graphql-go) server supporting react-relay.
```
Comment (sogko) [inline README.md]: I think you meant to write: `https://github.com/graphql-go/graphql`

**Comment by sogko** [comment]
Hi @pyros2097 

Thanks again for working on this.

This looks good 👍🏻

One thing, I haven't got around to set up CI for this project. Until that gets done, could you verify that all is okay by running the tests manually, and sharing the test output here? 

```bash
go test ./...
```

Cheers!

**Comment by pyros2097** [comment]
Yep ... I ran all the tests
```
ok  	github.com/graphql-go/graphql-relay-go	1.064s
ok  	github.com/graphql-go/graphql-relay-go/examples/starwars	0.035s
```

**Comment by sogko** [comment]
Alright, cool! Thanks again! ⭐️

**Pull request merged**